examples/echo/README.md

```diff
@@ -23,7 +23,8 @@ USERNAME=YOUR_DOCKER_HUB_USERNAME
 ```
 # tagging
 docker run --rm -v "$PWD":/app treeder/bump patch
-docker tag $USERNAME/func-echo:latest $USERNAME/func-echo:`cat VERSION`
+docker tag iron/func-echo:latest $USERNAME/func-echo:`cat VERSION`
+docker tag $USERNAME/func-echo:`cat VERSION` $USERNAME/func-echo:latest
 
```

Comment:
Actually, fn tool now automatically tags latest, so probably this is redundant.

#639

Reply:
I see.
I'll undo the changes.
Thanks.

Reply:
Thanks for the update!

Actually, now that latest tag is added automatically, we can update the docs to remove adding latest tag.

Would you mind updating to latest fn tool, confirming this behavior, and removing all reference to add latest tags from the documentation?

Reply:
Yeah, I'll work on it.

You mean documents of examples should have this kind of "fn" style?
https://github.com/iron-io/functions/blob/0c3206deb5f8ebda5283662a271138a84ef8ecb1/examples/hello/go/README.md

Reply:
yes, if possible !